RELEASES.md

```diff
@@ -784,6 +784,7 @@ Language
 - [You can now use `#[repr(transparent)]` on univariant `enum`s.][68122] Meaning
   that you can create an enum that has the exact layout and ABI of the type
   it contains.
+- [It is no longer disallowed to apply an attribute procedural macro to a module.][64273]
```

Comment:
Just to avoid a double negative which can be harder to read. Also, it is limited to inline-modules, and outer-attributes only.

```suggestion
- [You can now use outer attribute procedural macros on inline modules.][64273]
```